index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <link rel="icon" type="image/svg+xml" href="/vite.svg"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Kuama Christmas</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Black+And+White+Picture&family=Jost:ital,wght@0,100..900;1,100..900&family=Mountains+of+Christmas:wght@400;700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+          rel="stylesheet">
+</head>
+<body>
+<div class="overflow-hidden bg-gradient-to-b from-red-500 to-red-700 christmas-scene relative">
+
+    <div class="hidden h-full w-full flex items-center justify-center absolute pointer-events-none">
```

Comment:
Since class `hide` and `flex` mutate the same CSS property and the main expected result is to hide the div I would do:

```suggestion
    <div class="hidden h-full w-full absolute pointer-events-none">
```

Also, is it really needed?
I thought it served as a slot for the 3D model, but the 3D model is added into the `main.ts` directly inside `christmas-scene`

Reply:
lol, that's debug code to prepare screenshots for social platforms :)